assets/model_monitoring/components/tests/unit/test_compute_feature_importance.py

```diff
@@ -60,18 +60,22 @@ def test_compute_categorical_features_integer(self, get_zipcode_data):
 
     def test_determine_task_type_classification(self, get_fraud_data, get_zipcode_data):
         """Test deteremine task type for classification scenario."""
-        task_type = determine_task_type("ISPROXYIP", get_fraud_data)
+        task_type = determine_task_type(None, "ISPROXYIP", get_fraud_data)
         assert task_type == "classification"
-
-        task_type = determine_task_type("zipcode", get_zipcode_data)
+        task_type = determine_task_type("invalid", "zipcode", get_zipcode_data)
+        assert task_type == "classification"
+        task_type = determine_task_type("Classification", "zipcode", get_zipcode_data)
         assert task_type == "classification"
 
     def test_determine_task_type_regression(self, get_fraud_data):
```

Comment:
Could you please also add the case to the overridden like, customer passed in regression while from infer type is classification